pnpm-lock.yaml

```diff
@@ -7795,4 +7795,4 @@ packages:
 
 settings:
   autoInstallPeers: true
-  excludeLinksFromLockfile: false
+  excludeLinksFromLockfile: false
```

Comment:
same, why this change?

Reply:
pnpm-lock which checks the code using prettier --check pnpm-lock.yaml this chance does not going to reflect any issue in website. Don't worry